app/Actions/CreateOrEditConfig.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Actions;
+
+use Illuminate\Support\Facades\File;
+
+class CreateOrEditConfig
+{
+    public function __invoke()
+    {
+        $configPath = config('home_dir') . '/.lambo';
+        $configFilePath = "{$configPath}/config.json";
+
+        if (File::exists($configFilePath)) {
+            app('console')->info("Opening existing config file: {$configFilePath}");
+            $this->editConfig($configFilePath);
+            return 0;
+        }
+
+        if (!File::exists($configPath)) {
+            File::makeDirectory($configPath);
+        }
+
+        File::put("$configFilePath", $this->configFileTemplate());
+        app('console')->info("Opening new config file {$configFilePath}");
+        $this->editConfig($configFilePath);
+    }
+
+    public function isMac()
```

Comment:
Extract to a helper?

Reply:
Yep. Will wait until you finish committing.